src/utils/user-helpers.ts

```diff
@@ -45,6 +45,8 @@ export const convertToCristinPerson = (user: FlatCristinPerson): CreateCristinPe
   names: [
     { type: 'FirstName', value: user.firstName },
     { type: 'LastName', value: user.lastName },
+    { type: 'PreferredFirstName', value: user.preferredFirstName ?? '' },
+    { type: 'PreferredLastName', value: user.preferredLastName ?? '' },
```

Comment:
Endte vel opp med å ikke bruke dette likevel, da vi ikke trengte å endre tilbake til `CristinPerson` ved lagring? Tenker i så fall det kan fjernes til vi får behov for det. Nå kan disse bli satt til tom streng som vi egentlig ikke ønsker, da (jeg tror) vi heller vil at elementet ikke skal være der i de tilfellene
```suggestion
```